pig/src/main/java/org/apache/zeppelin/pig/PigInterpreter.java

```diff
@@ -107,9 +108,12 @@ public InterpreterResult interpret(String cmd, InterpreterContext contextInterpr
           return new InterpreterResult(Code.ERROR, ExceptionUtils.getStackTrace(e));
         }
       }
+      if (e.getCause() instanceof ParseException) {
+        return new InterpreterResult(Code.ERROR, e.getMessage());
```

Comment:
should `e.getMessage()` be `e.getCause().getMessage()`? or `ExceptionUtils.getStackTrace(e)`?